src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsConfig.java

```diff
@@ -164,6 +223,20 @@ private static XmlFile getConfigFile() {
     }
 
     /**
+     * Fixes JENKINS-11514: When {@code MaskPasswordsConfig.xml} is there but was created from
+     * version 2.8 (or older) of the plugin, {@link #globalVarPasswordPairs} can actually be
+     * {@code null} ==> Always use this getter to avoid NPEs.
+     *
+     * @since 2.9
+     */
+    private List<VarMaskRegex> getGlobalVarMaskRegexesList() {
+        if(globalVarMaskRegexes == null) {
```

Comment:
I'm slightly confused by this. The docs for [Collections.emptyList()](https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#emptyList()) say that it returns an immutable list. Wouldn't that be a problem when new regexes are added to this (e.g. in L139, ``getGlobalVarMaskRegexesList().add(varMaskRegex);``) ?